sdk/samples/iot/iot_sample_common.h

```diff
@@ -149,19 +154,26 @@ typedef enum
 
 extern bool is_device_operational;
 
+/**
+ * @brief Initialize an #iot_sample_error_log which holds error logging.
+ *
+ * @param[out] out_error_log A pointer to a #out_error_log instance to initialize.

```

Comment:
should out params be at the end of the param list?

Reply:
init functions are the exception